### ISSUE TITLE
not resistant against adversarial mangled names

### ISSUE THREAD

**Issue by whitequark** [Vector35/binaryninja-api#914]
To reproduce:
```
$ cat >test.c
void _ZN8iloveyouIS_S_S_S_S_EIS0_S0_S0_S0_S0_EIS1_S1_S1_S1_S1_EIS2_S2_S2_S2_S2_EIS3_S3_S3_S3_S3_EIS4_S4_S4_S4_S4_EIS5_S5_S5_S5_S5_EIS6_S6_S6_S6_S6_EIS7_S7_S7_S7_S7_EIS8_S8_S8_S8_S8_EIS9_S9_S9_S9_S9_EE() {}
int main() {}
$ cc test.c -o test
$ binaryninja test
```

**Comment by whitequark** [comment]
btw IDA guards against this https://twitter.com/marcan42/status/952854246126362624

**Comment by plafosse** [comment]
Fixed in build 1.1.1114
